api/client/stats_helpers.go

```diff
@@ -24,6 +24,7 @@ type containerStats struct {
 	NetworkTx        float64
 	BlockRead        float64
 	BlockWrite       float64
+	PidsCurrent      uint64
```

Comment:
I'm also planning on adding `PidsLimit` to the stats as well so we can get the cool `current / limit` stuff all the other cgroups do. Obviously that'll take a while (libcontainer + engine-api + docker).